README.md

```diff
@@ -1,40 +1,78 @@
 # React Components
 
 ## Objectives
-
 1. Explain how to create components using `React.createClass()`
 2. Explain how to create components by extending the `React.Component` class
 
-## Overview
+## Components?
+Now that we know how to render basic elements in React, let's kick things up a notch. Let's start by learning about
+React components!
+
+While elements are very basic... _elements_ of an application (for example, a `<div>`), a React application usually
```

Comment:
Do you mean html elements here? It's not totally clear. At first, I though you were referring to the difference between rendering an html element versus rendering another component. Do you need the comparison, though? Maybe just: "A React application usually considers of several components combined together. Unliked a simple html element, a React component is smarter and bigger... etc."?